requirements.txt

```diff
@@ -1 +1,2 @@
 PyQt5==5.12.2
+pyln-client>=0.8.0
```

Comment:
>=0.8.2 maybe ?

Reply:
Hi @darosior,

True, I put the >= because I think that the pip3 with > take the last version available on a public repository. Fix me if I'm saying something wrong. For this reason, I put the >= because give the opportunity to install the last (maybe) version available, in other words, is my idea work, we don't need more to update the dependence :)

Reply:
My bad i made a typo, i meant `>=`. This was about the version number.

Reply:
Yep, my dyslexia wins again (I did see the 2 in your message :smile:). Yep absolutely true the version is 0.8.2. I will fix it.

Reply:
@darosior done